cadquery/occ_impl/assembly.py

```diff
@@ -290,3 +294,145 @@ def toJSON(
         rv.extend(toJSON(child, loc, color, tolerance))
 
     return rv
+
+
+def toSimplified(assy: AssemblyProtocol, assy_name: str) -> TDocStd_Document:
+    """
+    Converts the assembly to a compound and saves that within the document
```

Comment:
Is this function actually working with deeply nested assys?

Reply:
No, it's not. I'll fix that before taking this out of draft.